node/pkg/logscribeconsumer/app.go

```diff
@@ -0,0 +1,220 @@
+package logscribeconsumer
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"os"
+	"time"
+
+	errorsentinel "bisonai.com/orakl/node/pkg/error"
+	"bisonai.com/orakl/node/pkg/utils/request"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
+
+func New(options ...AppOption) (*App, error) {
+	consoleWriter := zerolog.ConsoleWriter{Out: os.Stdout}
+
+	c := &AppConfig{
+		StoreInterval:     DefaultLogStoreInterval,
+		Buffer:            DefaultBufferSize,
+		LogscribeEndpoint: DefaultLogscribeEndpoint,
+		Level:             DefaultMinimalLogStoreLevel.String(),
```

Comment:
🤔 I think it'll work same even if you remove this default setting from config since you're catching error and setting level with this value again from line 41